pkg/webhook/clusterset/validatingWebhook.go

```diff
@@ -81,13 +81,21 @@ func (a *AdmissionHandler) validateResource(request *v1.AdmissionRequest) *v1.Ad
 			}
 			return status
 		}
-		//allow create/update legacy clusterset
-		if clusterset.Spec.ClusterSelector.SelectorType == clusterv1beta1.LegacyClusterSetLabel {
+		//allow create/update global clusterset
+		if clusterset.Name == clustersetutils.GlobalSetName {
```

Comment:
I think you have the validation on create, can we reuse the same logic?

Reply:
> I think you have the validation on create, can we reuse the same logic?

The logic here is used for both creating and updating a set.

Reply:
> if so, the message seems incorrect when create?

Update the message.